lib/phx_tools_web/components/core_components.ex

```diff
@@ -21,6 +21,36 @@ defmodule PhxToolsWeb.CoreComponents do
   @type rendered :: Rendered.t()
   @type selector :: String.t()
 
+  @doc """
+  Renders a card for being able to select OS
+  """
+  attr :id, :string, required: true
+  attr :href, :string, required: true
+  attr :os_icon, :string, required: true
+  attr :current_operating_system, :string, required: true
+  attr :os_name, :string, required: true
+
+  @spec os_link_card(assigns()) :: rendered()
+  def os_link_card(assigns) do
+    ~H"""
+    <.link navigate={@href}>
```

Comment:
Use `patch` instead of `navigate` for the same LiveView process.
https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#link/1
You'll need to use `handle_params` instead of `mount` to assign the OS.